pkgs/by-name/mi/mini-redis/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mini-redis";
+  version = "0.4.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-vYphaQNMAHajod5oT/T3VJ12e6Qk5QOa5LQz6KsXvm8=";
+  };
+
+  cargoHash = "sha256-oGyJxNzJX7PwMkDoT9Tb3xF0vWgQwuyIjKPgEkbPKyI=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
+
+  doCheck = false;
```

Comment:
If the checkPhase is disabled (my guess is it accesses networking or something), then could you add [versionCheckHook](https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook)?

Reply:
[non-blocking] You could also try testers.runCommand if you want. But I would say it's required.